podman/Containerfile

```diff
@@ -15,6 +15,19 @@ ARG PACKER_BUILD_NAME=
 ENV AI_PATH=/usr/src/automation_images \
     CONTAINER=1
 
+ARG IMG_SFX=
+ARG CIRRUS_TASK_ID=
+ARG GIT_HEAD=
+# Ref: https://github.com/opencontainers/image-spec/blob/main/annotations.md
+LABEL org.opencontainers.image.url="https://cirrus-ci.com/task/${CIRRUS_TASK_ID}"
+LABEL org.opencontainers.image.documentation="https://github.com/containers/automation_images/blob/${GIT_HEAD}/README.md#container-images-overview-step-2"
+# Renovate is sensitive to the following, they must point at the repo. and reflect commited
```

Comment:
"repo" is a standard term, not an abbreviation. Would you mind removing the period? Every time I see repo-with-a-period I think it's the end of a sentence, and then the next sentence makes no sense.

Reply:
I'll just remove this comment, it's not needed.